hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -33,6 +33,7 @@
 
 import operator
 import pandas
+import numpy as np
```

Comment:
Could you please do not rename numpy?

Reply:
why not? This is how numpy is use is everywhere else.

Reply:
@fschlimb to keep clear in the code what exactly used.

Reply:
I understand and I also generally prefer fully qualified names. In this particular case we should stay with common practice (e.g. import numpy as np and import pandas ad pd)